src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  background-color: yellowgreen;
+}
+
+.container {
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+img {
```

Comment:
Use classes instead 
![image](https://github.com/mate-academy/layout_moyo-header/assets/99403198/7eb9d89d-e61f-43c0-92b9-79105737d0fa)

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3775/files#r1310085429
didn't fix it